sources/packages/backend/libs/integrations/src/sfas-integration/sims-sfas-files/sims-to-sfas-restriction-record.ts

```diff
@@ -35,7 +35,7 @@ export class SIMSToSFASRestrictionRecord extends SIMSToSFASBaseRecord {
     record.append(this.recordTypeCode);
     record.appendNumberWithFiller(this.studentId, 10);
     record.appendNumberWithFiller(this.restrictionId, 10);
-    record.append(this.restrictionCode);
+    record.appendWithEndFiller(this.restrictionCode, 10, " ");
```

Comment:
Please use the `appendStringWithFiller`. The method will take care of the filler and append with end filler.

![image](https://github.com/user-attachments/assets/f6f11ade-c77f-49b5-8438-cd4fa4931f3a)